### ISSUE TITLE
[BUG] One or both stats files could not be read...

### ISSUE THREAD

**Issue by richb-hanover** [Fail-Safe/sqm-autorate#54]
**Script Version** "0.2.1" - current checkout from **testing/lua-threads**

**Describe the bug**
No rate control - `tc -s qdisc | grep bandwidth` remains at original SQM setting

**To Reproduce**
1. use  wget to run the uninstall.sh script
2. use the wget to run the setup.sh script. I used:
   `sh -c "$(wget -q -O- https://raw.githubusercontent.com/Fail-Safe/sqm-autorate/testing/lua-threads/sqm-autorate-setup.sh)"`
3. enable & start the sqm-autorate service
3. Examine `tc -s qdisc...` (below)
4. Examine `/tmp/sqm-autorate.log` (below)
5. Despite the `Rx stats file found!` (and Tx stats) message...
6. ... log file filled with `One or both stats files could not be read. Skipping rate control algorithm.`
5. Also note that there's no output to `/tmp/sqm-autorate.csv`

**Expected behavior**
`tc -s qdisc` should show changing bandwidth values
`/tmp/sqm-autorate.csv` should show changing values

**OpenWrt Details (please paste your output from `cat /etc/os-release`):**
 - OS-Release: 21.02.1
 - Connection Type + Speed: 25/25 symmetric

```
root@D-Link-DIR-878:~# tc -s qdisc | grep bandw
qdisc cake 8009: dev wan root refcnt 2 bandwidth 28Mbit besteffort triple-isolate nonat nowash no-ack-filter split-gso rtt 100ms noatm overhead 18
qdisc cake 800a: dev ifb4wan root refcnt 2 bandwidth 30001Kbit besteffort triple-isolate nonat wash no-ack-filter split-gso rtt 100ms noatm overhead 18
```

```
root@D-Link-DIR-878:~# head -n 40 /tmp/sqm-autorate.log
Starting sqm-autorate.lua v0.2.1
[INFO - 20220104T11:59:42]: Rx stats file found! Continuing...
[INFO - 20220104T11:59:42]: Tx stats file found! Continuing...
[INFO - 20220104T11:59:42]: Reflector Pool Size: 187
[WARN - 20220104T11:59:43]: One or both stats files could not be read. Skipping rate control algorithm.
[WARN - 20220104T11:59:44]: One or both stats files could not be read. Skipping rate control algorithm.
[WARN - 20220104T11:59:44]: One or both stats files could not be read. Skipping rate control algorithm.
[WARN - 20220104T11:59:45]: One or both stats files could not be read. Skipping rate control algorithm.
[WARN - 20220104T11:59:45]: One or both stats files could not be read. Skipping rate control algorithm.
[WARN - 20220104T11:59:46]: One or both stats files could not be read. Skipping rate control algorithm.
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 185.222.222.222 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 186.148.128.86 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 200.99.138.103 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 139.134.5.51 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 61.46.244.211 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 200.167.191.114 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 185.95.218.43 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 114.130.5.5 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 45.133.105.123 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 51.79.144.245 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 46.227.200.54 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 159.89.120.99 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 83.137.41.9 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 81.175.38.1 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 115.127.19.129 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 80.67.169.12 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 9.9.9.11 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 176.103.130.131 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 186.194.224.82 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 85.93.212.2 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 1.33.184.230 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 208.67.222.123 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 168.126.63.1 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 81.149.21.254 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 84.200.69.80 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 170.210.83.110 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 141.0.102.202 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 94.247.43.254 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 217.24.255.134 - skipping
[INFO - 20220104T11:59:46]: No data found from candidate reflector: 205.171.2.65 - skipping
root@D-Link-DIR-878:~#
```

**Comment by Fail-Safe** [comment]
@richb-hanover Would you say this has been resolved in v0.3.0 and can be closed?

**Comment by richb-hanover** [comment]
Yes, let's close this. 0.3.0 seems to be working
